ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[ssh_connection]
+pipelining = True
```

Comment:
I'm not sure that this one is working as expected when running within the container. I keep having the error like 
```
"Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user (rc: 1, err: chmod: invalid mode: 'A+user:hdbadm:rx:allow'\nTry 'chmod --help' for more information."
```

Reply:
Which playbook did this fail on?